src/utils/Utils.js

```diff
@@ -93,6 +93,16 @@ class Utils {
         });
     });
   }
+
+  /**
+   * Prints a deprecation warning for deprecated ChainSetup methods.
+   * See {@link https://github.com/OpenSTFoundation/mosaic.js/issues/57}.
+   */
+  static deprecationNoticeChainSetup(method) {
+    console.warn(
+      `${method} has been deprecated. See https://github.com/OpenSTFoundation/mosaic.js/issues/57 for migration instructions.`,
+    );
+  }
```

Comment:
you could also add
```js
  static deprecationNotice(method, issueNumber) {
    console.warn(
      `${method} has been deprecated. See https://github.com/OpenSTFoundation/mosaic.js/issues/${issueNumber} for migration instructions.`,
    );
  }
```
And then:
```js
static deprecationNoticeChainSetup(method) {
  const issueNumber = '57';
  deprecationNotice(method, issueNumber);
}
```

Reply:
Let's align. I also did https://github.com/OpenSTFoundation/mosaic.js/pull/87/files#diff-59f878307e3895d714d1a8af5c189d00R85-R94